.github/workflows/approve-trigger.yml

```diff
@@ -2,16 +2,25 @@ name: Approve Command Trigger
 on:
   issue_comment:
     types: [created]
+
 jobs:
   approve-command-trigger:
```

Comment:
We can add `if` conditions at the level of the job itself. For example, you can implement the conditional check on the issue's label as done here: https://github.com/ersilia-os/ersilia-self-service/blob/bd9203cc3370bd405d11dc99b09737c147c5223e/.github/workflows/self-service.yml#L8

Reply:
@DhanshreeA Okay. Will make these changes. Thank you